twitter/src/test/java/com/hazelcast/jet/contrib/twitter/TwitterSourceTest.java

```diff
@@ -187,12 +197,56 @@ public void testBatch() {
         }
     }
 
-    private static Properties loadCredentialsFromEnv() {
+    private static Properties loadCredentials() {
         Properties credentials = new Properties();
-        credentials.put("consumerKey", System.getenv("JET_TWITTER_CONNECTOR_CONSUMER_KEY"));
-        credentials.put("consumerSecret", System.getenv("JET_TWITTER_CONNECTOR_CONSUMER_SECRET"));
-        credentials.put("token", System.getenv("JET_TWITTER_CONNECTOR_TOKEN"));
-        credentials.put("tokenSecret", System.getenv("JET_TWITTER_CONNECTOR_TOKEN_SECRET"));
+        String consumerKey = System.getenv("JET_TWITTER_CONNECTOR_CONSUMER_KEY");
+        String consumerSecret = System.getenv("JET_TWITTER_CONNECTOR_CONSUMER_SECRET");
+        String token = System.getenv("JET_TWITTER_CONNECTOR_TOKEN");
+        String tokenSecret = System.getenv("JET_TWITTER_CONNECTOR_TOKEN_SECRET");
+
+        if (consumerKey != null || consumerSecret != null | token != null || tokenSecret != null) {
+            credentials.put("consumerKey", consumerKey);
+            credentials.put("consumerSecret", consumerSecret);
+            credentials.put("token", token);
+            credentials.put("tokenSecret", tokenSecret);
+            System.out.println("Twitter credentials are loaded from environment variables");
+        } else {
+            try {
+                credentials.load(Thread.currentThread().getContextClassLoader()
+                                       .getResourceAsStream("twitter-security.properties"));
+                System.out.println("Twitter credentials are loaded from twitter-security.properties file");
```

Comment:
Do not print to system.out, we do not care where the credentials are loaded from